manifest.json

```diff
@@ -17,23 +17,29 @@
 
     "content_scripts": [
         {
-            "matches": ["https://*/*/pull/*",
```

Comment:
Was this change in an effort to attempt to make the content script reload if the URL changed via ajax? I'm guessing it didn't work if so...